app/frontend/components/pages/osbl/show/GestionTab.tsx

```
@@ -7,16 +7,19 @@ import {
   CarouselPrevious
 } from '@/components/ui/carousel'
 import type { NewOsbl } from '@/pages/Contribution/types'
-import PieChartComponent from '@/components/charts/PieChartComponent'
+import PieChartComponent from '@/components/pages/osbl/show/GestionTab/PieChartComponent'
 import MyCheckbox from '@/components/shared/MyCheckbox'
 import { Separator } from '@/components/ui/separator'
 import { formatAmount } from '@/lib/formatters'
-
+import { Infinity } from 'lucide-react'
```

Biome (1.9.4)

[error] Do not shadow the global "Infinity" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)